.travis.yml

```diff
@@ -13,8 +13,10 @@ before_install:
   - chmod +x pom.xml
   
 script:
-  - mvn -DsomeModule.test.excludes="**/FlowTest.java, **/FlowTest2.java, **/TrafficGeneratedTest.java, **/FlowTestExternalComposition.java" test
+  - mvn -DsomeModule.test.excludes="**/FlowTest.java, **/FlowTest2.java, **/TrafficGeneratedTest.java, **/FlowTestExternalComposition.java, **/FlowTestTestExecution.java, **/FlowSourceChangeObject.java" test
   - mvn -DsomeModule.test.includes="**/FlowTest.java" test
   - mvn -DsomeModule.test.includes="**/FlowTest2.java" test
   - mvn -DsomeModule.test.includes="**/TrafficGeneratedTest.java" test
-  - mvn -DsomeModule.test.includes="**/FlowTestExternalComposition.java" test
\ No newline at end of file
+  - mvn -DsomeModule.test.includes="**/FlowTestExternalComposition.java" test
+  - mvn -DsomeModule.test.includes="**/FlowSourceChangeObject.java" test
+  - mvn -DsomeModule.test.includes="**/FlowTestTestExecution.java" test
```

Comment:
"FlowTestTestExecution.java", isn't it enough with one "Test" word in file name? -> "FlowTestExecution.java"